### PR TITLE
Fix spacemacs-purpose-popwin name

### PR DESCRIPTION
**Pull request by TheBB**
https://github.com/syl20bnr/spacemacs/issues/7439

### CODE REVIEW

**Comment by TheBB** [comment]
Is in.

File: layers/+spacemacs/spacemacs-purpose/local/spacemacs-purpose-popwin/spacemacs-purpose-popwin.el [hunk 1]
```diff
@@ -240,5 +240,5 @@ variables are updated instead."
     (ad-update 'popwin:create-popup-window)))
 (add-hook 'pupo-mode-hook #'pupo/sync-advices)
 
-(provide 'purpose-popwin)
+(provide 'spacemacs-purpose-popwin)
 ;;; purpose-popwin.el ends here
```
Comment (bmag) [inline layers/+spacemacs/spacemacs-purpose/local/spacemacs-purpose-popwin/spacemacs-purpose-popwin.el]: Missed the "ends here" line, and also the very first line in the file ("purpose-popwin.el --- [...]").

File: layers/+spacemacs/spacemacs-purpose/packages.el [hunk 2]
```diff
@@ -83,11 +83,11 @@
       (add-hook 'purpose-mode-hook #'spacemacs/window-purpose-sync-popwin)
       (spacemacs/window-purpose-sync-popwin))))
 
-(defun spacemacs-purpose/init-purpose-popwin ()
+(defun spacemacs-purpose/init-spacemacs-purpose-popwin ()
   (use-package spacemacs-purpose-popwin
     ;; don't load spacemacs-purpose-popwin if popwin is excluded.
     ;; can't wrap `spacemacs-purpose/init-purpose-popwin' in a top-level `when'
```
Comment (bmag) [inline layers/+spacemacs/spacemacs-purpose/packages.el]: Not related to this PR, but while we're at it we can delete these 2 comments as well.

**Comment by TheBB** [comment]
Thanks @bmag, I just pushed those fixes manually.

**Pull request merged**